modular_skyrat/master_files/code/datums/components/tippable.dm

```diff
@@ -0,0 +1,22 @@
+// Modular solution for alternative tipping visuals, for silicon/cyborgs.
```

Comment:
```suggestion
// Allows silicons/cyborgs to have unique icons when tipped over
```
With the original comment it took me a solid 5 minutes to understand what this code was for; I know you just copied the old one but while we're hereeeeee might as well

Reply:
Ok, I added it in commit 62e3911362371cde6126b311c3fd436f665c33b9